### ISSUE TITLE
Unable to run plotHighestExpression with dgCMatrix stored in counts slot of a SCE object

### ISSUE THREAD

**Issue by dweemx** [alanocallaghan/scater#49]
When a dgCMatrix is stored in the counts slot of a SingleCellExperiment and I try to run plotHighestExpression, there seems to be a problem when converting a dgCMatrix to data.frame:
```
 Error in as.data.frame.default(x[[i]], optional = TRUE) : 
  cannot coerce class "structure("dgCMatrix", package = "Matrix")" to a data.frame 
stop(gettextf("cannot coerce class \"%s\" to a data.frame", deparse(class(x))), 
    domain = NA) 
as.data.frame.default(x[[i]], optional = TRUE) 
as.data.frame(x[[i]], optional = TRUE) 
data.frame(data) 
setNames(data.frame(data), value.name) 
melt.default(df_pct_exprs_by_cell) 
reshape2::melt(df_pct_exprs_by_cell) 
plotHighestExprs(object, ...) 
plotQC(sce.cleaned, type = "highest-expression")
```

**Comment by LTLA** [comment]
Is this with the BioC-release version? The BioC-devel version? Or the Github version?

**Comment by dweemx** [comment]
BioC-release version (3.6)
Version of scater in my environment: scater_1.6.1

**Comment by LTLA** [comment]
This should be fixed in version 1.6.3. You can either wait a few days for it to build on the BioC system, or you can get it directly with `devtools::install_github("davismcc/scater@RELEASE_3_6")`.
